.github/workflows/ci.yml

```diff
@@ -227,7 +243,9 @@ jobs:
             -DIREE_BUILD_COMPILER=OFF \
             -DIREE_ENABLE_LLD=ON \
             -DIREE_ENABLE_RUNTIME_TRACING=ON \
-            -DIREE_TRACING_PROVIDER=${TRACING_PROVIDER}
+            -DIREE_TRACING_PROVIDER=${TRACING_PROVIDER} \
+            -DIREE_HAL_DRIVER_CUDA=ON \
+            -DIREE_HAL_DRIVER_HIP=ON 
```

Comment:
Yep - we _do_ want these enabled in the tracing jobs. We've caught a few issues where features were not correctly guarded by `#ifdef` in these drivers.

Also like my other comment, this looks like it could get merged into the `runtime` job using CMake presets. Not in this PR though.